nvflare/fuel/utils/dict_utils.py

```diff
@@ -12,18 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
This update(d, u) is a useful function. Better to keep it even we are not directly use it right now.